cdap-e2e-tests/src/e2e-test/features/namespaceadmin/NSAdminDesignTime.feature

```diff
@@ -0,0 +1,115 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+@Namespaceadmin
+Feature: NameSpaceAdmin - Validate nameSpace admin design time scenarios
+
+  @Namespaceadmin
+  Scenario:Verify user is able to click the nameSpace admin tab and successfully navigates to the page
+    Given Open Datafusion Project to configure pipeline
+    When Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Verify that the user is navigated to nameSpace admin page successfully
+
+  @Namespaceadmin
+  Scenario:Validate user is able to open compute profile page and select a provisioner
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Click on create profile button for "default" Namespace
+    Then Select a provisioner: "existingDataProc" for the compute profile
+    Then Click on close button of compute profile properties page
```

Comment:
Are we not validating if we are able to successfully select the provisioner ?

Reply:
Added the validation steps .